src/gui/DatabaseWidget.cpp

```diff
@@ -901,12 +938,39 @@ void DatabaseWidget::unlockDatabase(bool accepted)
     }
 }
 
+/**
+ * @author Fonic <https://github.com/fonic>
+ * Add 'copy-on-doubleclick' functionality for certain columns
+ *
+ * TODO:
+ * If pull request #1298 gets merged, double-clicking column 'Attachments'
+ * could open the new details view (see second screenshot)
```

Comment:
This PR was merged, please implement this

Reply:
Now or in a follow up?

Reply:
now would be perfect

Reply:
I'll have to defer this as well, as the review process has already eaten up much more time than anticipated.